bin/check-elb-sum-requests.rb

```diff
@@ -142,11 +130,7 @@ def check_sum_requests(elb)
   end
 
   def run
-    @message = if elbs.size == 1
-                 elbs.first.inspect
-               else
-                 "#{elbs.size} load balancers total"
-               end
+    @message = "#{elbs.size} load balancers total"
```

Comment:
Since we are removing the check to be grammatically correct we should add a parenthesis around the plural form
> balancer(s)

Reply:
Noted, will change it, thanks.